dev/io.openliberty.jakarta.restfulWS.3.1.api_fat/fat/src/io/openliberty/jakarta/restfulWS31api/fat/app/canload/ApiTestServlet.java

```diff
@@ -0,0 +1,207 @@
+        assertTrue(canLoad("jakarta.ws.rs.PathParam"));
+        assertTrue(canLoad("jakarta.ws.rs.POST"));
+        assertTrue(canLoad("jakarta.ws.rs.Produces"));
+        assertTrue(canLoad("jakarta.ws.rs.PUT"));
+        assertTrue(canLoad("jakarta.ws.rs.QueryParam"));
+    }
+
+    @Test
+    public void testCanLoadJakartaWsRsClientClasses() throws Exception {
+        // interfaces
+        assertTrue(canLoad("jakarta.ws.rs.client.AsyncInvoker"));
+        assertTrue(canLoad("jakarta.ws.rs.client.Client"));
+        assertTrue(canLoad("jakarta.ws.rs.client.ClientRequestContext"));
+        assertTrue(canLoad("jakarta.ws.rs.client.ClientRequestFilter"));
+        assertTrue(canLoad("jakarta.ws.rs.client.ClientResponseContext"));
+        assertTrue(canLoad("jakarta.ws.rs.client.ClientResponseFilter"));
+        assertTrue(canLoad("jakarta.ws.rs.client.CompletionStageRxInvoker"));
+        assertTrue(canLoad("jakarta.ws.rs.client.Invocation"));
+        assertTrue(canLoad("jakarta.ws.rs.client.Invocation$Builder"));
+        assertTrue(canLoad("jakarta.ws.rs.client.InvocationCallback"));
+        assertTrue(canLoad("jakarta.ws.rs.client.RxInvoker"));
+        assertTrue(canLoad("jakarta.ws.rs.client.RxInvokerProvider"));
+        assertTrue(canLoad("jakarta.ws.rs.client.SyncInvoker"));
+        assertTrue(canLoad("jakarta.ws.rs.client.WebTarget"));
+        // classes
+        assertTrue(canLoad("jakarta.ws.rs.client.ClientBuilder"));
+        assertTrue(canLoad("jakarta.ws.rs.client.Entity"));
+        // exceptions
+        assertTrue(canLoad("jakarta.ws.rs.client.ResponseProcessingException"));
+    }
+
+    @Test
+    public void testCanLoadJakartaWsRsContainerClasses() throws Exception {
+        // interfaces
+        assertTrue(canLoad("jakarta.ws.rs.container.AsyncResponse"));
+        assertTrue(canLoad("jakarta.ws.rs.container.CompletionCallback"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ConnectionCallback"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ContainerRequestContext"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ContainerRequestFilter"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ContainerResponseContext"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ContainerResponseFilter"));
+        assertTrue(canLoad("jakarta.ws.rs.container.DynamicFeature"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ResourceContext"));
+        assertTrue(canLoad("jakarta.ws.rs.container.ResourceInfo"));
+        assertTrue(canLoad("jakarta.ws.rs.container.TimeoutHandler"));
+        // annotations
+        assertTrue(canLoad("jakarta.ws.rs.container.PreMatching"));
+        assertTrue(canLoad("jakarta.ws.rs.container.Suspended"));
+    }
+
+    @Test
+    public void testCanLoadJakartaWsRsCoreClasses() throws Exception {
+        // interfaces
+        assertTrue(canLoad("jakarta.ws.rs.core.Configurable"));
+        assertTrue(canLoad("jakarta.ws.rs.core.Configuration"));
```

Comment:
`assertTrue(canLoad("jakarta.ws.rs.core.EntityPart"));` for the new Multipart handling

Reply:
Done.  And there were a few more I added as well. Thanks